frontend/src/metabase/visualizations/components/ChartSettings/BaseChartSettings/BaseChartSettings.unit.spec.tsx

```diff
@@ -28,7 +28,7 @@ function widget(widget: Partial<Widget> = {}): Widget {
     section: "section-" + Math.random(),
     props: {},
     ...widget,
-  };
+  } as Widget;
```

Comment:
this is for a test and uses an ID that's not in visualization settings, so there's a bunch of type issues otherwise.